ssl/ssl_sess.c

```diff
@@ -26,54 +26,41 @@ static int remove_session_lock(SSL_CTX *ctx, SSL_SESSION *c, int lck);
 
 DEFINE_STACK_OF(SSL_SESSION)
 
-__owur static int sess_timedout(time_t t, SSL_SESSION *ss)
+__owur static ossl_inline OSSL_TIME time_t_to_ossl_time(time_t t)
 {
-    /* if timeout overflowed, it can never timeout! */
-    if (ss->timeout_ovf)
-        return 0;
-    return t > ss->calc_timeout;
+    OSSL_TIME ot;
+
+    ot.t = t;
+    ot.t *= OSSL_TIME_SECOND;
```

Comment:
Probably should use ossl_ticks2time

Reply:
??? If it's moved to time.h, it's okay as is I think.

Reply:
Yeah ok